.github/workflows/trigger-publish.yml

```diff
@@ -39,4 +39,30 @@ jobs:
           REPO: ${{ toJSON('temporal') }}
           BRANCH: ${{ toJSON(steps.get_branch.outputs.branch) }}
         run: |
+          # Dispatch the workflow
           curl -fL -X POST -H "Accept: application/vnd.github.v3+json" -H "Authorization: token $PAT" "https://api.github.com/repos/$PARENT_REPO/actions/workflows/$WORKFLOW_ID/dispatches" -d '{"ref":'"$PARENT_BRANCH"', "inputs": { "repo":'"$REPO"', "branch":'"$BRANCH"' }}'
+          
+          # Wait a few seconds for the workflow to start
+          sleep 10
+          
+          # Get the latest workflow run for this workflow
+          run_id=$(curl -fL -H "Accept: application/vnd.github.v3+json" -H "Authorization: token $PAT" "https://api.github.com/repos/$PARENT_REPO/actions/workflows/$WORKFLOW_ID/runs" | jq -r '.workflow_runs[0].id')
```

Comment:
I think we need either `set -euo pipefail` on top so this fails if the call fails, or check whether `run_id` is empty